### PR TITLE
mobile menu links

### PR DESCRIPTION
**Pull request by varffs**

### CODE REVIEW

File: header.php [hunk 1]
```diff
@@ -76,30 +76,31 @@
...
+            'taxonomy' => 'project_type',
+            'field' => 'id',
+            'terms' => $type->term_id
+          )
+        ),
+        'posts_per_page' => 1,
+        'post_type' => 'project'
+      ));
+?>
+          <a class="only-mobile" href="<?php echo get_term_link($type); ?>"><li><?php echo $type->name; ?></li></a>
```
Comment (mixcas) [inline header.php:98]: This is semantically wrong. `ul` can't contain other than `li`

Comment (varffs) [inline header.php:98]: well 1: it was like that before. 2: we do this often. 3: do we need to add a width 100% css rule and put the links inside?

Comment (mixcas) [inline header.php:98]: why bother using a `ul` then?
we shouldn't be doing this if we care about semantics. i just noticed it happens in other parts of the site. Also `ul` inside another `ul` should be wrapped with a `li`.

its ok if u wanna leave it like that. but i consider we should avoid it in future projects.

Comment (varffs) [inline header.php:98]: ok merge this and we avoid in the future. is there any actual reason to use lists for menus? i guess that is semantic? but unless everything is nested equal level lists for main and submenus are never going to be semantic...

**Pull request merged**